### ISSUE TITLE
Update Wiki on benchmarking with various providers

### ISSUE THREAD

**Issue by kk-min** [vhive-serverless/STeLLAR#301]
The Wiki needs to be updated once `feature-serverless-framework-deployment` is merged into main. Some notes on things that can be updated/included:

- How the user interfaces with the tool (experiment JSON file)
- How to deploy using various providers (preparing raw-code specific to providers, env variables etc.)
- Obtaining results

**Comment by kk-min** [comment]
Closed as documentation for all supported providers on `main` has been added to the Wiki.
